src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -118,8 +120,11 @@ see the following articles written by community members:
 
 [A deep dive into Flutter's accessibility widgets]: {{site.medium}}/flutter-community/a-deep-dive-into-flutters-accessibility-widgets-eb0ef9455bc
```

Comment:
This link is still not working

Reply:
Oy vey. The failure was due to the "wrong kind" of apostrophe in the article's title. It's fixed now.